src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,190 @@
+        avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    />
+</React.Fragment>
+
+Modal dialogs are always blocking — they must be dismissed before the user can interact with the rest of the application content, typically by clicking a dismiss / close button or by clicking elsewhere on the screen.
+
+> If you are collecting a lot of information, it may be better to move this form to a separate page in your application rather than trying to place too much content into a modal.
 
 ## Sidebar
 
-The most common example of a sidebar is the application navigation menu. Typically, this opens on the side of the screen and is used to navigate to the different areas of the application.
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[SidePanel]}
+        caption={'Sidebars can be used to show additional information about a selected item.'}
+    />
+    <MaterialDesignDescription title={'Sheets: Side'} url={'https://material.io/components/sheets-side'} />
+</React.Fragment>
 
-Sidebars are also used extensively for displaying information about selected items in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item). They can also present simple form data and inputs (such as radio buttons, checkbox, dropdown menus, etc.) for editing a selected item. This type of sidebar should be temporary such that the user can easily close it with a close button or clicking elsewhere on the screen.
+Sidebars (Side Sheets in the Material documentation) are used to present supplemental information to users. Most commonly, they display information about a selected item in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item). However, they can also be used standalone, such as the legend for a chart or map.
 
-### Modal vs. Sidebar
+<ImageGrid regularWidth images={[TrendLegend]} caption={'A permanent sidebar can be used for chart / map legends.'} />
 
-In general, modals and sidebars can be used together to set up and display complex data. When building your application, you may be faced with the question of "Should I use a modal or a sidebar for this?" In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data. In some circumstances, you may need a full-page overlay (sidebar or bottomsheet) to accomplish more complex tasks.
+They can also present simple form data and inputs, such as radio buttons, checkbox, dropdown menus, etc. for editing a selected item. This type of sidebar should be dismissible, such that the user can easily close it with a close button or clicking elsewhere on the screen.
 
-## Bottom Sheet
+Sidebars should always appear on the opposite side of the screen as the main [navigation](/patterns/navigation).
```

Comment:
Would like to see how this side bar behaves on mobile

Reply:
(Either stacked, open a temporary side bar, or do a new page sliding in from the right)

Reply:
If that is too much to research for, feel free to hide this section for now

Reply:
I think that for anything that is not explicitly mentioned, we defer to Material guidelines

Reply:
Sounds good. I was thinking about the stacked view for data visualization, but maybe we can cover that in data viz pattern